guides/v2.2/cloud/project/project-conf-files_services-redis.md

```diff
@@ -16,18 +16,18 @@ We support Redis versions 3.2 and 5.0. See [Configure Redis]({{ page.baseurl }}/
 {:.procedure}
 To enable Redis:
 
-1. Add the required name and type to the `.magento/services.yaml` file.
+1. Add the required name, type, and version to the `.magento/services.yaml` file.
```

Comment:
Per the main services topic, the two values required are `name` and `type`. The name value is the relationship used in configuration file and can be anything you want as long as it is consistently used. The type value actually includes two values: service name and service version. In the SQL topic, we say add the name, type, and disk value. These are meant to refer to service values defined on the main services topic.